### PR TITLE
Update to Version 7.2.0

### PR DESCRIPTION
**Pull request by rxm7706**
Python >=3.7 (Match Source)
Updated dependencies from source
Added rxm7706 as co-maintainer
Rendered using Grayskull

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@guzman-raphael - please review and merge at your convenience to fix the feedstock and publish the latest version. 
Linked Issue - https://github.com/conda-forge/minio-feedstock/issues/1
Thanks and regards.

**Comment by rxm7706** [comment]
@conda-forge/core 
@guzman-raphael 

Please review and merge at your convenience to fix the feedstock and publish the latest version. I have included myself as a maintainer, and am happy to help keep the feedstock updated.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,31 +1,34 @@
...
-    - python-dateutil
-    - configparser
+    - urllib3
+    - argon2-cffi
+    - pycryptodome
+# Not found via grasykull, Not required for python >=3.7 https://github.com/minio/minio-py/blob/7.2.0/setup.py
+    #- future
+    #- pytz
+    #- python-dateutil
+    #- configparser
```
Comment (hmaarrfk) [inline recipe/meta.yaml]:
```suggestion
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,31 +1,34 @@
...
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b0b687c1ec9be422a1f8b04c65fb8e43a1c090f9508178db57c434a17341c404
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/minio-{{ version }}.tar.gz
+  sha256: 4b015b018d10c1505f7c3e724fa7c2267760ac7bee6463a624cbf22cd272877b
 
 build:
   noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```
Comment (hmaarrfk) [inline recipe/meta.yaml]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps
```
I think we set a global environment variable for this.

Comment (hmaarrfk) [reply recipe/meta.yaml]: I did try to find our reference for this, but yeah, fewer flags the better until things break.

**Comment by hmaarrfk** [comment]
thanks!

**Pull request merged**